This patch is for festival for Debian GNU/Linux, to ensure
manual includes functions/variables from festival library

Index: b/lib/festdoc.scm
===================================================================
--- a/lib/festdoc.scm
+++ b/lib/festdoc.scm
@@ -55,6 +55,7 @@
 
  ;; Need to ensure all library files are actually loaded if they contain
  ;; funcstions/variables which have to be put in the manual
+ (require 'festival)
  (require 'display)
  (require 'mbrola)
  (require 'tilt)
